lib/models/lrs.js

```diff
@@ -58,10 +83,14 @@ schema.statics.updateStatementCount = async (lrs) => {
   });
 };
 
-schema.statics.decrementStatementCount = async (lrs) => {
+/**
```

Comment:
I think we need to change the way this function is called in "lib/services/lrs.js" since it seems to be passing the full model, although we might be able to delete the `decrementStatementCount` function from that file since I don't think it's used anymore.

Reply:
Oh, cool! I didn’t even notice it, thanks!

Reply:
Done!